### PR TITLE
Proj4 review & new quickstart

### PR DESCRIPTION
**Pull request by julien2512**
Proj4 overview update
Adding a quickstart for proj4 as is (no more metacrs) from http://proj4.org/

### CODE REVIEW

**Comment by cvvergara** [comment]
Hello @julien2512 

I had a fast look.
To me, it looks "portable" between versions.

Maybe adding links to the subsections, like this: 

```
 * Use the `proj`_ tool
 * Use the `cs2cs`_ tool
 * Use the `geod`_ tool
```
and/or

This instruction looks like it generates contents of the page automatically.
```
.. contents:: Contents
```
like in here:
https://live.osgeo.org/en/quickstart/udig_quickstart.html

**Comment by camerons** [comment]
Hi Julien, I've only skimmed what you have written so far, but looks like your quickstart is on the right path. I'd normally suggest adding images, but I assume that everything is text based, so images are not needed.
For the Project Overview, I think that needs to be rewritten. At the moment, it is written as a "MetaCRS Project Overview" and I think it needs to be a "Proj4 Project Overview". Ie, write it as if you are only talking about Proj4, and call the document proj4_overview.rst 
It probably should include a sentence: "Proj4 has been translated from C to a number of other languages including <a href="...">proj4js</a>, ...
Have a look at the other library write ups on OSGeo-Live for ideas on layout. Also look at the comments in the PostGIS Project Overview (which our reference overview) for tips on what should be included.
Thanks for your help. Cameron

**Comment by kalxas** [comment]
Hi @julien2512 and thank you for your contribution.
In addition to the above comments, we will need an e-mail that you agree with the license.
Please send something like this:
"Hi all, I'd like to contribute XXX to OSGeo-Live. I confirm that my contributions to OSGeo-Live will be compatible with the OSGeo-Live license guidelines at the time of contribution."
as mentioned in: https://wiki.osgeo.org/wiki/Live_GIS_Disc#Git_migration
Can you also add your name in the contributors.csv file as part of this PR?

**Comment by julien2512** [comment]
Hi all,

I'd like to contribute proj.4 to OSGeo-Live. I confirm that my
contributions to OSGeo-Live will be compatible with the OSGeo-Live license
guidelines at the time of contribution.

Regards,
Julien
P.S.: I will also add my name in the contributors.csv file as part of my PR
with the other changes requested !

2017-06-18 18:12 GMT+02:00 Angelos Tzotsos <notifications@github.com>:

**Comment by kalxas** [comment]
Thanks @julien2512 
Any thoughts on the Proj4 project overview as @camerons described?
We will retire the MetaCRS one for this release...
You have my +1 for the proj4 quickstart

**Comment by julien2512** [comment]
Thanks Julien :)

Cheers,

Nicolas

**Comment by julien2512** [comment]
My PR update is for this evening. I got my smartphone only for now !

Regards,
Julien

Le 19 juin 2017 15:51, "Angelos Tzotsos" <notifications@github.com> a
écrit :

**Comment by julien2512** [comment]
@cvvergara 
> This instruction looks like it generates contents of the page automatically.
>   .. contents:: Contents

Done.
And thank you for the good trick !

@camerons 
>  call the document proj4_overview.rst

Done.

**Comment by julien2512** [comment]
@camerons
> For the Project Overview, I think that needs to be rewritten. At the moment, it is written as a "MetaCRS Project Overview" and I think it needs to be a "Proj4 Project Overview". Ie, write it as if you are only talking about Proj4

Done.

@camerons
> It probably should include a sentence: "Proj4 has been translated from C to a number of other languages including proj4js, ...

Done.

**Comment by julien2512** [comment]
@camerons 
> Have a look at the other library write ups on OSGeo-Live for ideas on layout. Also look at the comments in the PostGIS Project Overview (which is our reference overview) for tips on what should be included.
I won't go further than proj.4 project himself !

@camerons 
> Thanks for your help.
You're welcome.

**Comment by julien2512** [comment]
@kalxas 
> Any thoughts on the Proj4 project overview as @camerons described?
> We will retire the MetaCRS one for this release...

Tell me if this update seems good for you !

File: ca/overview/overview.rst [hunk 1]
```diff
@@ -116,7 +116,7 @@ Llibreries Geospacials
 * :doc:`jts_overview` - Java Topology Suite 
 * :doc:`geotools_overview` - Java GIS Toolkit  
 * :doc:`geos_overview` - C/C++ Spatial Library  
-* :doc:`metacrs_overview` - Coordinate Reference System Transformations  
+* :doc:`Proj.4 <proj4_overview>` - Coordinate Reference System Transformations (MetaCRS)
```
Comment (kalxas) [inline ca/overview/overview.rst]: this just needs to be:
* :doc:`proj4_overview` - Coordinate Reference System Transformations

File: de/overview/overview.rst [hunk 2]
```diff
@@ -120,7 +120,7 @@ GIS Bibliotheken
 * :doc:`jts_overview` - Java Topology Suite
 * :doc:`geotools_overview` - Java GIS Toolkit 
 * :doc:`geos_overview` - C/C++ Spatial Library 
-* :doc:`metacrs_overview` - Coordinate Reference System Transformations 
+* :doc:`Proj.4 <proj4_overview>` - Coordinate Reference System Transformations (MetaCRS)
```
Comment (kalxas) [inline de/overview/overview.rst]: same as above

File: en/overview/overview.rst [hunk 3]
```diff
@@ -115,7 +115,7 @@ Geospatial Libraries
 * :doc:`jts_overview` - Java Topology Suite
 * :doc:`geotools_overview` - Java GIS Toolkit 
 * :doc:`geos_overview` - C/C++ Spatial Library 
-* :doc:`metacrs_overview` - Coordinate Reference System Transformations 
+* :doc:`Proj.4 <proj4_overview>` - Coordinate Reference System Transformations (MetaCRS)
```
Comment (kalxas) [inline en/overview/overview.rst]: same

**Comment by julien2512** [comment]
@kalxas no problemo

**Comment by kalxas** [comment]
thanks :)

File: en/overview/overview.rst (hunk 3 above)
Comment (julien2512) [inline en/overview/overview.rst]: done

File: de/overview/overview.rst (hunk 2 above)
Comment (julien2512) [inline de/overview/overview.rst]: done

File: ca/overview/overview.rst (hunk 1 above)
Comment (julien2512) [inline ca/overview/overview.rst]: done

**Pull request merged**